ecosystem/sep-0006.md

```diff
@@ -131,7 +131,7 @@ Request Parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | The code of the asset the user wants to deposit with the anchor. Ex BTC, ETH, USD, INR, etc. This may be different from the asset code that the anchor issues. Ex if a user deposits BTC and receives MyBTC tokens, `asset_code` must be BTC.
+`asset_code` | string | The code of the asset the user wants to deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```

Comment:
💡 I think it is helpful to continue to include the last example, it at least makes it clearer to me:
```suggestion
`asset_code` | string | The code of the asset the user wants to deposit with the anchor. For example, if a user deposits BTC and receives MyBTC tokens, `asset_code` will be BTC. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```